lib/easypost/models/error.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# EasyPost Error object.
+class EasyPost::Models::Error
+  attr_reader :code, :field, :message
+
+  # Initialize a new EasyPost Error
+  def initialize(code, field = nil, message = nil)
+    @code = code
+    @field = field
+    @message = message
+  end
+
+  def self.from_api_error_response(data)
```

Comment:
issue: methods should describe what they do, this name could be clarified and get a docstring

Reply:
Resolved